versioned_docs/version-v2.18.x/user-guide/api-mediation/configuration-client-certificates.md

```diff
@@ -3,9 +3,9 @@
 :::info Roles: system programmer, system administrator, security administrator
 :::
 
-You can authenticate against API ML onboarded APIs. This functionality is disabled by default. Follow the steps in this article to enable authentication against API ML onboarded APIs. 
+In Zowe, you can authenticate against API ML onboarded APIs. This functionality is disabled by default. Follow the steps in this article to enable authentication against API ML onboarded APIs. 
 
-There are two methods to enable client certificate functionality: 
+There are two methods to enable x509 client certificate functionality: 
```

Comment:
Even though the certificates we use are x509 we also have x509 as the authentication scheme. 

For the authentication scheme you need neither setups. For the client certificate authentication in general you need either one of them. 

So maybe something like:
  There are two methods to enable client certificate functionality integrated with the SAF: